Add documentation for Recombee destination (#7049)
segmentio/segment-docs: src/connections/destinations/catalog/recombee-ai/index.md

---
title: Recombee AI Destination
rewrite: true

> maintenance: true

Reviewer: ```suggestion
maintenance: true
maintenance-content: This destination is no longer available in the Segment catalog but will remain active in workspaces where it has already been installed. Recombee has developed an updated destination built on the Actions framework. See [Recombee Destination](/docs/connections/destinations/catalog/actions-recombee/) for more information.
```

---
title: Recombee AI Destination
rewrite: true
maintenance: true

> maintenance-content: This destination is no longer available in the Segment catalog but will remain active in workspaces where it has already been installed. Recombee has developed an updated destination built on the Actions framework. See [Recombee Destination](/docs/connections/destinations/catalog/actions-recombee/) for more information.

Reviewer 2: ```suggestion
maintenance-content: This destination is no longer available in the Segment catalog, but will remain active in workspaces where it has already been configured. Recombee has developed an updated destination built on the Actions framework. See [Recombee Destination](/docs/connections/destinations/catalog/actions-recombee/) for more information.
```

hide-settings: true
hide-personas-partial: true
id: 6095391bd839b62fca8a8606
versions:
  - name: Recombee (Actions)
    link: /docs/connections/destinations/catalog/actions-recombee
---

[Recombee](https://recombee.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} is a Recommender as a Service that can use your data to provide the most accurate recommendations of content or products for your users.

> Use this Segment destination to send your interaction data (views, purchases, plays, etc.) to Recombee.

Reviewer 2: ```suggestion
Use this Segment destination to send your interaction data, like views, purchases, or plays, to Recombee.
```
